examples/python/helloworld/async_greeter_client_with_options.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""gRPC Python AsyncIO helloworld.Greeter client with channel options and timeout parameters."""
+
+import asyncio
+import logging
+
+import grpc
+
+import helloworld_pb2
+import helloworld_pb2_grpc
+
+
+async def run() -> None:
+    # For more channel options, please see https://grpc.io/grpc/core/group__grpc__arg__keys.html
+    async with grpc.aio.insecure_channel(target='localhost:50051',
+                                         options=[('grpc.lb_policy_name',
```

Comment:
Nit: This might look a little cleaner if you pull the options out to a constant.

Reply:
Good idea. Promoted to a constant.